Data-Structures/Vectors/Vector2.js

```diff
@@ -4,75 +4,6 @@
  * https://en.wikipedia.org/wiki/Vector_(mathematics_and_physics)).
```

Comment:
Up here, I reformatted the documentation block a bit. But maybe that's just me being overly obsessive about code style ;)

Reply:
I absolutely agree and so I also copied here. But I noticed, in your code, you forgot to remove a closing parenthesis (the very last) which corrupted the URL, so I just quickly fixed that.